fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1008,6 +1009,27 @@ public void rebuildIndex(@Nonnull FDBRecordStore store) {
         return buildAsync(this::buildEndpoints, false, additionalLogMessageKeyValues);
     }
 
+    /**
+     * Stop any ongoing online index build (only if it uses synchronized sessions) by forcefully release
```

Comment:
```suggestion
     * Stop any ongoing online index build (only if it uses synchronized sessions) by forcefully releasing
```

And "synchronized sessions" there should possibly be a link

Reply:
Also `releasing` instead of `release`.